source/_integrations/qnap.markdown

```diff
@@ -3,124 +3,31 @@ title: QNAP
 description: Instructions on how to integrate the QNAP sensor within Home Assistant.
 ha_category:
   - System Monitor
-ha_release: 0.38
+ha_release: 2022.10
 ha_iot_class: Local Polling
 ha_domain: qnap
+ha_config_flow: true
 ha_platforms:
   - sensor
 ha_integration_type: integration
 ---
 
-This `qnap` sensor allows getting various statistics from your [QNAP NAS](https://www.qnap.com/en-us/).
+The QNAP integration allows getting various statistics from your [QNAP NAS](https://www.qnap.com/en-us/).
 
 ## Configuration
```

Comment:
The configuration header is already provided by the include below. This causes to have a duplicate configuration header at this point. Did you check the output?